packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue

```diff
@@ -13,6 +13,10 @@
 				<template v-for="(tab, index) in tabs" :key="index">
 					<component :is="tab" v-show="selectedViewIndex === index" />
 				</template>
+
+				<section v-if="slots.shared">
+					<slot name="shared" />
+				</section>
```

Comment:
dev can add a shared section so that sections that are shared between tabs can be written once

Reply:
I find the naming of "shared" slightly weird because we also have the notions of sharing projects and assets, if in all our design sketches this right-pane is used for "preview" would it be better to call it as such?

Also @Jami159 and @Tom-Szendrey should be tagged to review here.

Reply:
preview works as well - I guess I was thinking more of a cases perhaps when a section needs to be shared that isn't a preview necessarily?  But I don't think there is a case where it the shared section wouldn't be shared so Ill rename as such